tests/flooding.rs

```diff
@@ -34,8 +34,8 @@ fn flood_request_propagation() {
             path_trace: vec![(100, NodeType::Client)],
         }),
         routing_header: SourceRoutingHeader {
-            hop_index: 1,
-            hops: vec![0, 1, 2, 3, 4],
+            hop_index: 0,
+            hops: vec![],
         },
```

Comment:
I would keep the original here so we can confirm that the drone clears the original header and sends an empty one